ethportal-api/src/types/execution/header.rs

```diff
@@ -557,6 +605,109 @@ mod tests {
             &hex_decode("0x17cf53189035bbae5bce5c844355badd701aa9d2dd4b4f5ab1f9f0e8dd9fea5b")
                 .unwrap(),
         );
+        assert_eq!(header.number, 17034871);
+        assert_eq!(header.hash(), expected_hash);
+    }
+
+    #[test]
+    fn dencun_rlp() {
```

Comment:
I think we can remove this test, it was mostly a helper test while debugging, and the following test covers the same functionality